gateway-ha/src/main/java/io/trino/gateway/ha/security/SessionCookie.java

```diff
@@ -3,7 +3,7 @@
 import jakarta.ws.rs.core.NewCookie;
 import jakarta.ws.rs.core.Response;
 
-public static final class SessionCookie
+public final class SessionCookie
```

Comment:
This change is unrelated to "Declare strings as static" commit. Also, I'm not sure why you removed `static`.